common/oslimits.go

```diff
@@ -0,0 +1,14 @@
+package common
```

Comment:
This file needs to be disabled on windows. You can put this line at the top of the file:

```
// +build !windows
```